gemfiles/rails60.gemfile

```diff
@@ -9,7 +9,7 @@ gem "rails", "~> 6.0.0"
 gem "carrierwave", "~> 2"
 gem "combustion", ">= 1.3"
 gem "rbnacl", ">= 6"
-gem "sqlite3"
+gem "sqlite3", "< 2"
```

Comment:
We had a patch here that locked these versions to `~> 1.4`, but it looks like the upstream updated their unconstrained use of sqlite3 to `< 2`, which is essentially the same, so I kept the upstream to minimize our changes.